spec/index.html

```diff
@@ -500,10 +500,14 @@ <h3>Blank nodes</h3>
 
     <p>Suppose I is a simple interpretation and A is a mapping from a set of blank nodes
       to the universe IR of I.
-      Define the mapping [I+A] to be I on <a>names</a>,
-      and A on blank nodes on the set: [I+A](x)=I(x) when x is a <a>name</a>
-      and [I+A](x)=A(x) when x is a blank node;
-      and extend this mapping to triples and RDF graphs using the rules given above for ground graphs.
+      Define the mapping [I+A] as </p>
+      <ul>
+	<li> [I+A](x)=I(x) when x is a <a>name</a>, </li>
+	<li> [I+A](x)=A(x) when x is a blank node, </li>
+	<li> [I+A](x)= RE( [I+A](x.s), [I+A](x.p), [I+A](x.o) )  where x.s, x.p, and x.o are the first, second, and third components of x, respectively. </li>
```

Comment:
```suggestion
	<li>When x is a <a>name</a>, [I+A](x)=I(x).</li>
	<li>When x is a blank node, [I+A](x)=A(x).</li>
	<li>Otherwise, [I+A](x)= RE( [I+A](x.s), [I+A](x.p), [I+A](x.o) ), where x.s, x.p, and x.o are the first, second, and third components of x, respectively.</li>
```

Reply:
It is conventional to have the result first and the condition second so I am not in favour of this change.

Reply:
What about simply changing WHEN to IF then?

Reply:
I've revised my suggested change.

Reply:
In logic and mathematics (which this is) multi-branch definitions are often written on multiple lines with each line having the value and then the condition for that value.

Reply:
Condition -> mismatch -> condition -> mismatch -> condition -> match -> action

makes infinitely more sense to me than

Action? -> Condition -> mismatch -> Action? -> condition -> mismatch -> Action? -> condition -> match -> go back and perform last action.